packages/cactus-plugin-ledger-connector-fabric-socketio/src/main/typescript/connector/sign-utils.ts

```diff
@@ -71,19 +70,8 @@ function sign(
   algorithm: string,
   keySize: number,
 ) {
-  const hashAlgorithm = algorithm.toUpperCase();
-  const hashFunction = hash[`${hashAlgorithm}_${keySize}`];
-  const ecdsaCurve = ellipticCurves[`p${keySize}`];
-  const ecdsa = new elliptic.ec(ecdsaCurve);
-  const key = jsrsa.KEYUTIL.getKey(privateKey) as any;
-
-  const signKey = ecdsa.keyFromPrivate(key.prvKeyHex, "hex");
-  const digest = hashFunction(proposalBytes);
-
-  let sig = ecdsa.sign(Buffer.from(digest, "hex"), signKey);
-  sig = preventMalleability(sig, key.ecparams);
-
-  return Buffer.from(sig.toDER());
```

Comment:
It looks like the only dependency to the old `fabric-client` library is a hash function mapping. There's some code in the Weaver SDK that does something similar to what the deleted code is doing: https://github.com/hyperledger/cacti/blob/main/weaver/sdks/fabric/interoperation-node-sdk/src/eciesCrypto.js#L186. Perhaps you can use this as a dependency (or just copy the code for now) instead of deleting the functionality altogether?